docs/site/src/content/docs/plugins/core/common-expressions.mdx

```diff
@@ -147,6 +147,30 @@ function replace(
 ): string;
 ```
 
+### split
+
+> Splits a string into an array of substrings based on a separator. Optionally limits the number of splits.
+
+```ts
+function split(
+  value: string,
+  separator: string,
+  limit?: number,
+): Array<string>;
+```
+
+### substr
```

Comment:
Potentially support `substring` as well, and maybe all the common JS standard lib functions

Reply:
https://github.com/player-ui/player/issues/672